client/src/components/ChatComponent/styles.scss

```diff
@@ -0,0 +1,12 @@
+.chat-box {
```

Comment:
@rollaball , I do think it's better not to go with `global css` at all.

Reply:
Updated. Using styled-components now.